docs/data-sources/environment.md

```diff
@@ -3,12 +3,12 @@
 page_title: "devcycle_environment Data Source - terraform-provider-devcycle"
 subcategory: ""
 description: |-
-  DevCycle Environment Data Source.
+  DevCycle Environment Data Source. Read data from a given DevCycle Environment. R
 ---
 
 # devcycle_environment (Data Source)
 
-DevCycle Environment Data Source.
+DevCycle Environment Data Source. Read data from a given DevCycle Environment. R
```

Comment:
All of these changes to the .md files were the result of running the `go generate` command